package.json

```diff
@@ -5,6 +5,9 @@
     "build:dev": "tsc --build",
     "clean": "lerna run clean",
     "install": "lerna bootstrap --hoist",
+    "prettier": "prettier --list-different --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}\"",
+    "prettier:check": "prettier --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}\"",
+    "prettier:files": "prettier --write",
```

Comment:
What does the `prettier:files` command do?

Reply:
@vidartf To be honest, I took the script I found in JupyterLab's package.json file. I have just removed `prettier:files`.